module/coriolis-roll.js

```diff
@@ -413,6 +456,150 @@ function getTooltipData(results) {
   return data;
 }
 
+function getActorType(rollData) {
+  return `${rollData.actorType}`;
+}
+
+function getRollType(rollData) {
+  return `${rollData.rollType}`;
+}
+
+function getRollAttribute(rollData) {
+  return `${rollData.attribute}`;
+}
+
+function getRollAttributeName(rollData) {
+  let AttrKey = "";
+  switch (rollData.attributeKey) {
+    case "strength":
+      return AttrKey = game.i18n.localize("YZECORIOLIS.AttrStrength");
+    case "agility":
+      return AttrKey = game.i18n.localize("YZECORIOLIS.AttrAgility");
+    case "wits":
+      return AttrKey = game.i18n.localize("YZECORIOLIS.AttrWits");
+    case "empathy":
+      return AttrKey = game.i18n.localize("YZECORIOLIS.AttrEmpathy");
+  }
+  return AttrKey;
+}
+
+function getRollSkill(rollData) {
+  return `${rollData.skill}`;
+}
+
+function getRollSkillName(rollData) {
+  let SkillKey = "";
```

Comment:
Couldn't we reuse `YZECORIOLIS.skills` .... like `YZECORIOLIS.skills[rollData.skillKey]`?

Reply:
Yes, I wasn't sure how that is reusable. But I tryed it now till it works. (easy if you understand it >__<)